tests/unit/fileupload.test.js

```diff
@@ -61,136 +55,11 @@ describe('FileUpload Plugin', () => {
-            expect(uploader.currentStep).toEqual('disclaimerScreen');
-            uploader.displayStep('uploadScreen');
-            uploader.toggleWidget();
-            uploader.toggleWidget();
-            expect(uploader.currentStep).toEqual('uploadScreen');
-            uploader.displayStep('successScreen');
-            uploader.toggleWidget();
-            uploader.toggleWidget();
-            expect(uploader.currentStep).toEqual('homeScreen');
-            uploader.displayStep('errorScreen');
-            uploader.toggleWidget();
-            uploader.toggleWidget();
-            expect(uploader.currentStep).toEqual('homeScreen');
-        });
-    });
-
-    describe('incoming events', () => {
-        test('systempatcher', () => {
-            const onSystemPatcherStatusEvent = jest.spyOn(uploader, 'onSystemPatcherStatusEvent');
-            const onSystemPatcherLastResultEvent = jest.spyOn(uploader, 'onSystemPatcherLastResultEvent');
-
-            instance.emit('systempatcher', 'status some various params');
-            expect(onSystemPatcherStatusEvent).toHaveBeenNthCalledWith(1, 'some various params');
-            expect(onSystemPatcherLastResultEvent).toHaveBeenCalledTimes(0);
-
-            instance.emit('systempatcher', 'last_result some various params');
-            expect(onSystemPatcherStatusEvent).toHaveBeenCalledTimes(1);
-            expect(onSystemPatcherLastResultEvent).toHaveBeenCalledTimes(1, 'some various params');
-
-            instance.emit('systempatcher', 'unrelevant some various params');
-            expect(onSystemPatcherStatusEvent).toHaveBeenCalledTimes(1);
-            expect(onSystemPatcherLastResultEvent).toHaveBeenCalledTimes(1);
-        });
-    });
-
-    describe('outgoing events', () => {
-        let sendEventSpy;
-
-        beforeEach(() => {
-            sendEventSpy = jest.spyOn(instance, 'sendEvent');
-        });
-
-        afterEach(() => {
-            sendEventSpy.mockRestore();
-        });
-
-        test('flash opengapps', () => {
-            uploader.displayStep('disclaimerScreen');
-            const button = document.getElementsByClassName('gm-upload-main-bottom-buttons-right')[0];
-
-            button.click();
-
-            expect(sendEventSpy).toHaveBeenCalledTimes(1);
-            expect(instance.outgoingMessages[0]).toEqual({
-                channel: 'systempatcher',
-                messages: ['install opengapps'],
-            });
-        });
-
-        test('cancel flash opengapps', () => {
-            uploader.displayStep('uploadScreen');
-            const button = document.getElementsByClassName('gm-upload-main-bottom-buttons-right')[0];
-
-            button.click();
-
-            expect(sendEventSpy).toHaveBeenCalledTimes(1);
-            expect(instance.outgoingMessages[0]).toEqual({
-                channel: 'systempatcher',
-                messages: ['cancel'],
-            });
-        });
-
-        test('reboot instance', () => {
```

Comment:
Could you explain (in the commit message would be very helpful): why we don't need these tests anymore?

Reply:
Thanks! It's helpful :)